space-invaders.asm

```diff
@@ -1,12 +1,65 @@
 ; constants
-%define NUM_INVADERS 12
+
+; GAME WINDOW PARAMETERS
 %define GAME_WIDTH 27
 %define GAME_HEIGHT 25
 
+; GAME PARAMETERES
+%define NUM_INVADERS 12
 %define INVADERS_MOVE_CYCLES 40
 %define INVADERS_SHOOT_CYCLES 4
 %define BULLETS_MOVE_CYCLE 28
 
+; SPECIAL CONSTANT
+%define INVALID_STATE 0x0000
+
+; ICONS
+%define PLAYER 'M'
```

Comment:
To further inhance readability, each constant should probably get a prefix that resembles the "type" of the constant.

```asm
%define ICON_PLAYER 'M'
%define ICON_INVADER 'T'
%define ICON_BULLET '|'
; ...
```

Reply:
Yes, that's good!